docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.md

```diff
@@ -0,0 +1,273 @@
+                                          "snowplow__session_identifiers",
+                                          "snowplow__session_lookback_days",
+                                          "snowplow__session_sql",
+                                          "snowplow__session_stitching",
+                                          "snowplow__session_timestamp",
+                                          "snowplow__start_date",
+                                          "snowplow__total_all_conversions",
+                                          "snowplow__upsert_lookback_days",
+                                          "snowplow__user_identifiers",
+                                          "snowplow__user_sql",
+                                          "snowplow__user_stitching_id"
+                                          ] },
+  { title: "Contexts, Filters, and Logs", fields: ["snowplow__app_id",
+                                                  "snowplow__enable_consent",
+                                                  "snowplow__enable_cwv",
+                                                  "snowplow__enable_iab",
+                                                  "snowplow__enable_ua",
+                                                  "snowplow__enable_yauaa",
+                                                  "snowplow__has_log_enabled",
+                                                  "snowplow__page_view_passthroughs",
+                                                  "snowplow__session_passthroughs",
+                                                  "snowplow__ua_bot_filter",
+                                                  "snowplow__user_first_passthroughs",
+                                                  "snowplow__user_last_passthroughs",
+                                                  ] },
+  { title: "Warehouse Specific", fields: ["snowplow__databricks_catalog",
+                                          "snowplow__entities_or_sdes",
+                                          "snowplow__page_view_context",
+                                          "snowplow__iab_context",
+                                          "snowplow__ua_parser_context",
+                                          "snowplow__yauaa_context",
+                                          "snowplow__cmp_visible_events",
+                                          "snowplow__consent_preferences_events",
+                                          "snowplow__browser_context","snowplow__session_context","snowplow__mobile_context","snowplow__geolocation_context","snowplow__application_context","snowplow__screen_context","snowplow__application_error_events","snowplow__screen_view_events","snowplow__deep_link_context",
+                                          "snowplow__enable_load_tstamp",
+                                          "snowplow__derived_tstamp_partitioned"] }
```

Comment:
Annoyingly will need to update them here too...

Reply:
it was fine there already but added cwv